workflow/exp/exp.conus12km

```diff
@@ -41,8 +41,8 @@ export LBC_UNGRIB_GROUP_TOTAL_NUM=1
 export LBC_GROUP_TOTAL_NUM=1
 
 export RETRO_PERIOD="2024052700-2024052800"
-export CYCLEDEF_IC_STEP_HRS=6   # do ic jobs every step hours
-export CYCLEDEF_LBC_STEP_HRS=6  # do lbc jobs every step hours
+export CYCLEDEF_IC_STEP_HRS=12  # do ic jobs every step hours
+export CYCLEDEF_LBC_STEP_HRS=12  # do lbc jobs every step hours
```

Comment:
Is it enough to do LBC tasks every 12 hours?
I think we need to do them every 6 hours.

Reply:
This is for controlled tests to ensure the pure effect from cycling. Please confirm @hu5970 .